pkg/apis/devops/v1alpha2/register.go

```diff
@@ -35,21 +39,199 @@ var (
 )
 
 func addWebService(c *restful.Container) error {
+
 	webservice := runtime.NewWebService(GroupVersion)
 
-	tags := []string{"devops"}
+	tags := []string{"DevOps"}
+
+	webservice.Route(webservice.GET("/devops/{devops}").
+		To(devopsapi.GetDevOpsProjectHandler).
+		Doc("get devops project").
+		Metadata(restfulspec.KeyOpenAPITags, tags).
+		Param(webservice.PathParameter("devops", "devops project's Id")).
+		Returns(http.StatusOK, "success", devops.DevOpsProject{}).
```

Comment:
use `success` as constant here.  In order to keep unify,  the description message suggest use `ok`, default is `ok`.